src/actions/LogActions.tsx

```diff
@@ -213,38 +221,47 @@ export function getLogOutput({ userMessage, send = true }: LogOutputOptions): Th
         logOutput.loggedInUser.actions.push(actionData)
       }
     }
-    logOutput.data += `App version: ${packageJson.version}
-App build: ${getVersion()}.${getBuildNumber()}
-os: ${Platform.OS} ${Platform.Version}
-device: ${getBrand()} ${getDeviceId()}
-`
+    const appInfo = `
+      App version: ${packageJson.version}
+      App build: ${getVersion()}.${getBuildNumber()}
+      os: ${Platform.OS} ${Platform.Version}
+      device: ${getBrand()} ${getDeviceId()}
+    `
```

Comment:
This indentation gets copied into the string literally, so it has to look like it did before, or the logs you upload to the server are going to be messed up. Please put it back.

Reply:
Indentation has been reverted